playbooks/common/openshift-cluster/upgrades/pre/verify_upgrade_targets.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: Verify containers are available for upgrade
   command: >
-    docker pull {{ openshift_cli_image }}:{{ openshift_image_tag }}
+    docker pull {{ openshift_cli_image }}
```

Comment:
Why are you removing openshift_image_tag?

Reply:
openshift_image_tag is already regex_replaced into openshift_cli_image via openshift_image_default via l_os_registry_url

https://github.com/openshift/openshift-ansible/pull/8447/files#diff-23ff377caf1c3db9840ce07eede4eda0R15